js/index.js

```diff
@@ -0,0 +1,135 @@
+const ham = document.getElementById('toggleButton');
+const close = document.getElementById('close');
+const title = document.getElementById('title');
+const men = document.getElementById('menu');
+const listItems = document.getElementsByClassName('list-item');
+const portfolio = document.getElementById('portfolio');
+
+const menuHandler = () => {
+  men.classList.toggle('menu-on');
+  close.classList.toggle('hide');
+  ham.classList.toggle('hide');
+  title.classList.toggle('hide');
+};
+
+ham.addEventListener('click', menuHandler);
+close.addEventListener('click', menuHandler);
+for (let i = 0; i < listItems.length; i += 1) {
+  listItems[i].addEventListener('click', menuHandler);
+}
+
+// Popup
+
+const projects = [
```

Comment:
- Kindly fix this style issue:
![Screen Shot 2021-10-07 at 16 08 00](https://user-images.githubusercontent.com/22258274/136462579-afeb045e-03b6-4c4c-b9b9-3f48000d00c4.png)

Reply:
- [ ] The issue here has not been properly resolved, please refer back to the Figma template chosen for this portfolio and design your popup in the same way.